### PR TITLE
Small info about AccessoryDescription and BodyPartDescription

### PR DESCRIPTION
**Pull request by karl-police**
## Changes

Adds a description for https://create.roblox.com/docs/en-us/reference/engine/classes/AccessoryDescription and https://create.roblox.com/docs/en-us/reference/engine/classes/BodyPartDescription with a little bit of consistency from how it's written in ``HumanoidDescription``. I didn't add much because I am not really familiar with the syntax, nor is it finished, but the sort of idea is already there, so 🤷 

## Checks

By submitting your pull request for review, you agree to the following:

- [x] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [x] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [x] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

File: content/en-us/reference/engine/classes/AccessoryDescription.yaml [hunk 1]
```diff
@@ -2,8 +2,10 @@ name: AccessoryDescription
 type: class
 category:
 memory_category: Instances
-summary: ''
-description: ''
+summary: |
+  Describes the appearance of an Accessory for the HumanoidDescription.'
```
Comment (totallytavi) [inline content/en-us/reference/engine/classes/AccessoryDescription.yaml:6]: Remove the `'` at the end of this line

File: content/en-us/reference/engine/classes/BodyPartDescription.yaml [hunk 2]
```diff
@@ -2,8 +2,10 @@ name: BodyPartDescription
 type: class
 category:
 memory_category: Instances
-summary: ''
-description: ''
+summary: |
+  Describes the appearance of a Body Part for the HumanoidDescription'
+description: |
+  **BodyPartDescription** is an object that stores the description for an Avatar's Body Parts, such as the Color. It is ment to be placed underneath a `Class.HumanoidDescription`, in order to work with `Class.Humanoid:ApplyDescription()`.
```
Comment (totallytavi) [inline content/en-us/reference/engine/classes/BodyPartDescription.yaml:8]: Typo: replace `ment` with `meant`

File: content/en-us/reference/engine/classes/AccessoryDescription.yaml [hunk 3]
```diff
@@ -2,8 +2,10 @@ name: AccessoryDescription
...
+description: |
+  **AccessoryDescription** is an object that stores the description for an `Class.Accessory`. It is ment to be placed underneath a `Class.HumanoidDescription`, in order to work with `Class.Humanoid:ApplyDescription()`.
```
Comment (totallytavi) [inline content/en-us/reference/engine/classes/AccessoryDescription.yaml:8]: Typo: replace `ment` with `meant`

File: content/en-us/reference/engine/classes/BodyPartDescription.yaml [hunk 4]
```diff
@@ -2,8 +2,10 @@ name: BodyPartDescription
 type: class
 category:
 memory_category: Instances
-summary: ''
-description: ''
+summary: |
+  Describes the appearance of a Body Part for the HumanoidDescription'
```
Comment (totallytavi) [inline content/en-us/reference/engine/classes/BodyPartDescription.yaml:6]: Remove the `'` at the end of this line

**Review by totallytavi — CHANGES_REQUESTED** [review]
I've left comments for stuff that flagged the GitHub Actions bot

**Review by totallytavi — APPROVED** [review]
LGTM

**Comment by Jozhuaheine123457** [comment]
Oh okay

**Review by Jozhuaheine123457 — CHANGES_REQUESTED** [review]
This need two change as long then u well be good

**Comment by karl-police** [comment]
uhhh?

File: content/en-us/reference/engine/classes/BodyPartDescription.yaml [hunk 5]
```diff
@@ -2,8 +2,10 @@ name: BodyPartDescription
 type: class
 category:
 memory_category: Instances
-summary: ''
-description: ''
+summary: |
+  Describes the appearance of a Body Part for the HumanoidDescription
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/classes/BodyPartDescription.yaml:6]:
```suggestion
  Describes the appearance of a `Enum. BodyPart` for the `Class.HumanoidDescription`.
```

File: content/en-us/reference/engine/classes/BodyPartDescription.yaml [hunk 6]
```diff
@@ -2,8 +2,10 @@ name: BodyPartDescription
...
+description: |
+  **BodyPartDescription** is an object that stores the description for an Avatar's Body Parts, such as the Color. It is meant to be placed underneath a `Class.HumanoidDescription`, in order to work with `Class.Humanoid:ApplyDescription()`.
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/classes/BodyPartDescription.yaml:8]:
```suggestion
  **BodyPartDescription** is an object that stores the description for an avatar's body parts, such as the `Class.BodyPartDescription.Color|Color`. It is meant to be placed underneath a `Class.HumanoidDescription` in order to work with `Class.Humanoid:ApplyDescription()`.
```

File: content/en-us/reference/engine/classes/AccessoryDescription.yaml [hunk 7]
```diff
@@ -2,8 +2,10 @@ name: AccessoryDescription
 type: class
 category:
 memory_category: Instances
-summary: ''
-description: ''
+summary: |
+  Describes the appearance of an Accessory for the HumanoidDescription.
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/classes/AccessoryDescription.yaml:6]:
```suggestion
  Describes the appearance of an `Class.Accessory` for the `Class.HumanoidDescription`.
```

File: content/en-us/reference/engine/classes/AccessoryDescription.yaml [hunk 8]
```diff
@@ -2,8 +2,10 @@ name: AccessoryDescription
...
+description: |
+  **AccessoryDescription** is an object that stores the description for an `Class.Accessory`. It is meant to be placed underneath a `Class.HumanoidDescription`, in order to work with `Class.Humanoid:ApplyDescription()`.
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/classes/AccessoryDescription.yaml:8]:
```suggestion
  **AccessoryDescription** is an object that stores the description for an `Class.Accessory`. It is meant to be placed underneath a `Class.HumanoidDescription` in order to work with `Class.Humanoid:ApplyDescription()`.
```

**Review by IgnisRBX — APPROVED** [review]

**Comment by IgnisRBX** [comment]
Thanks for the contribution! I made a few tiny edits and linked up to other API pages.

**Pull request merged**